capstone/static/js/search/main.vue

```diff
@@ -237,6 +236,7 @@
 
               self.results[self.page] = results_json.results;
             })
+            .then(self.scrollToResults)
```

Comment:
@jcushman I think this does everything we want it to do— It won't run if you hit the throw, but .then(self.stopLoading) does.